raiden/log_config.py

```diff
@@ -24,6 +27,42 @@ def _get_log_handler(formatter, log_file):
         }
 
 
+def redactor(blacklist: Dict[Pattern, str]) -> Callable:
+    """Returns a function which transforms a str, replacing all matches"""
+    def processor_wrapper(msg: str) -> str:
+        for regex, repl in blacklist.items():
+            if repl is None:
+                repl = '<redacted>'
+            msg = regex.sub(repl, msg)
```

Comment:
Not a big deal, but could be improved a bit: Using `.sub()` with the regex will replace the whole expression (including the `access_?token` in the case below).

Reply:
That's why `repl` starts with`\1`, to put it back there. It was the most generic and functional thing I noticed, to be able to either remove any unwanted information or keep only part of it (through the repl in the value in the `blacklist` dict).

Reply:
Ok missed that, then it's fine.